docs/guides/upgrade-from-open.mdx

```diff
@@ -23,7 +23,7 @@ If necessary, follow the steps in the [Set up to use IBM Cloud](./cloud-setup#cl
 <span id="update-cloud-account"></span>
 ## Upgrade your IBM Cloud account
 
-Skip this if you already have an IBM Cloud account with a Pay-As-You-Go Plan. All new IBM Cloud accounts are set up with a Pay-As-You-Go Plan.
+Skip this if you already have an IBM Cloud account with a Pay-as-you-go plan. All new IBM Cloud accounts are set up with a Pay-as-you-go plan.
```

Comment:
is that maybe referring to the [account type](https://cloud.ibm.com/docs/account?topic=account-accounts) `Pay-As-You-Go` instead of a "plan"?